sdk/storage/azure-storage-common/inc/azure/storage/common/access_conditions.hpp

```diff
@@ -58,4 +58,22 @@ namespace Azure { namespace Storage {
     Azure::Core::Nullable<std::string> LeaseId;
   };
 
+  /**
+   * @brief Specifies HTTP options for conditional requests based on ContentHash.
+   */
+  struct ContentHashAccessConditions
+  {
+    /**
+     * @brief Specify this header to perform the operation only if the resource's ContentHash
+     * matches the value specified.
+     */
+    Azure::Core::Nullable<ContentHash> IfMatch;
```

Comment:
Is this easily confused with ETag match? How .Net SDK name this variable?

Reply:
I believe they are missing this feature: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Files.Shares/src#L3668

BTW, it is difficult to be confused since customers would need to set `options.SourceContentHashAccessCondition.IfMatch`.

Reply:
I also mentioned this in another comment